kwalitee/config.py

```diff
@@ -276,8 +276,12 @@
 HOOK_TEMPLATE = """#!/usr/bin/env python
 # -*- coding: utf-8 -*-
 import sys
-from kwalitee import create_app
-from kwalitee.hooks import {hook}
+try:
+    from invenio_kwalitee import create_app
```

Comment:
please note that you have to change the hook since this package has changed its name to `kwalitee`.